src/mixxx.cpp

```diff
@@ -1438,14 +1440,13 @@ void MixxxMainWindow::slotViewFullScreen(bool toggle) {
         // Fix for "No menu bar with ubuntu unity in full screen mode" Bug
         // #885890 and Bug #1076789. Before touching anything here, please read
         // those bugs.
+        QApplication::setAttribute(Qt::AA_DontUseNativeMenuBar, true);
```

Comment:
In my tests I did exactly that (just at the beginning of `createMenuBar()`, not directly before the call) and I did set it in main.cpp, not in mixxx.cpp.

So I wonder how that makes the difference...
Anyway, crash fixed!